### ISSUE TITLE
Further rewrite of `Gameplay/Multiplayer`

### ISSUE THREAD

**Issue by The-Last-Cookie** [ppy/osu-wiki#7165]
This article was rewritten in [#6817](https://github.com/ppy/osu-wiki/pull/6813), but not all tasks have been resolved.

These are the tasks that are still open:
- [ ] move article from `Gameplay/Multiplayer` to `Client/Interface/Multiplayer`
- [ ] game mechanics for tag coop probably need to be tested, because this article seems to document some bugs
- [ ] check no video game modier trivia (see [this comment](https://github.com/ppy/osu-wiki/pull/6813#discussion_r833740091))
- [ ] update outdated images
	- especially `Multi_team_vs.jpg`, `Multi_tag_co-op.jpg` and `Multi_tag_team_vs.jpg`

**Comment by Soetch** [comment]
So is there still a need to move the Multiplayer folder ? Cleanup was done in #6817.

**Comment by The-Last-Cookie** [comment]
Yes, it still needs to be moved. See [this](https://github.com/ppy/osu-wiki/pull/6813#issuecomment-1024429314) and [this comment](https://github.com/ppy/osu-wiki/pull/6813#pullrequestreview-931737437).

As I don't plan on further editing the English version, you may take on one or more of the above tasks if you want.

**Comment by Soetch** [comment]
> Yes, it still needs to be moved. See [this](https://github.com/ppy/osu-wiki/pull/6813#issuecomment-1024429314) and [this comment](https://github.com/ppy/osu-wiki/pull/6813#pullrequestreview-931737437).
> 
> As I don't plan on further editing the English version, you may take on one or more of the above tasks if you want.

Alright then, I'm moving it and taking the updating-image one. Thanks for replying !

**Comment by Gamingzone123** [comment]
Just thowing it out there, this along with many other articles will need an update when lazer becomes the stable branch

**Comment by FeeFort** [comment]
> game mechanics for tag coop probably need to be tested, because this article seems to document some bugs

if by that you mean this bug:
`However, in Tag-team VS, when a team's health drops to zero, even though the [health bar](/wiki/Client/Interface/Health_bar) will be refilled, the match will judge them as *failed* and end immediately.`
then we checked this, and it works exactly as written, what can we do with this?

> check no video game modifier trivia (see [this comment](https://github.com/ppy/osu-wiki/pull/6813#discussion_r833740091))

we downloaded 2 old builds `b20121203` and `b20121223`
in `b20121203` no video mod was still present and in multiplayer players could individually choose this mod, even if they weren't the host.
in `b20121223` no video mod was removed and visual settings panel was implemented. in multiplayer mods menu was empty for non-hosts.
and this part of article:
`- No Video used to be the *only* game modifier allowed for players to use, but it was removed in favour of its placement in the [visual settings](/wiki/Client/Interface/Visual_settings).`
is correct.

**Comment by TicClick** [comment]
closing as per above
